### PR TITLE
vsc manual

### PR DESCRIPTION
**Pull request by Krejza9**
### Description
Add manuals for  visual studio code
### Related Issue

Closes

### CODE REVIEW

**Review by Tony931023 — CHANGES_REQUESTED** [review]

File: docs/vsc_requirements.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+| OpenSLL ≥ 1.0.x              |
+| CMake ≥ 3.16                 |
+| MS Visual Studio Build Tools |
+
+1. Git Extensions
+   
+   1. You will need Git installed. You can get the latest version here https://git-scm.com/download/win
+   
+   
+
```
[the remark this answers is not in the document]
Comment (Krejza9) [reply docs/vsc_requirements.md]: done

File: docs/vsc_requirements.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+	1. Install extensions
+		1. Click **View**->**Extensions**
+			
+			<a href="../images/vsc_extensions.png.png" target="_blank"><img src="../images/vsc_extensions.png"></a>
+		1. In Search bar type **C/C++ Extension Pack**
+		1. Click on green install button	
+			
+			<a href="../images/visualstudio_tools.png" target="_blank"><img src="../images/vcs_extension_pack_install.png" height="50%" width="50%"></a>
+
+
```
[the remark this answers is not in the document]
Comment (Krejza9) [reply docs/vsc_requirements.md]: done

File: docs/windows-vsc-core-installation.md [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+**libcrypto-1_1-x64.dll** → C:\OpenSSL-Win64\bin
+
+#### About compilation log and report
+
+pdb files only exist if you compile on Debug or RelWithDebInfo modes, it's not mandatory but it's recommended to compile core on at least RelWithDebInfo mode to get proper crashlogs. If you compile on Release mode the pdb files aren't needed.
+
+To report crash logs it's MANDATORY to compile on Debug or RelWithDebInfo mode.
+
+<br>
+
```
[the remark this answers is not in the document]
Comment (Krejza9) [reply docs/windows-vsc-core-installation.md]: done

**Comment by Kitzunu** [comment]
I will look at this when I get time in the weekend.

**Comment by Kitzunu** [comment]
I'll have to review this at a later date wont have time this weekend

**Pull request merged**